doc/source/cluster/kubernetes/configs/ray-cluster.log.yaml

```diff
@@ -12,8 +12,12 @@ data:
         Path_Key true
         Refresh_Interval 5
     [OUTPUT]
-        Name stdout
+        Name loki
         Match *
+        Host loki-gateway
+        Port 80
+        Labels RayCluster_name=raycluster-complete-logs
```

Comment:
Can we use K8s labels (https://docs.fluentbit.io/manual/pipeline/filters/kubernetes) instead to dynamically set up the log label?